konrad/ozone.py

```diff
@@ -28,6 +28,10 @@
     >>>     ozone_model(atmosphere=atmosphere, timestep=...)
     >>> final_ozone_profile = atmosphere['O3'][-1]
 
+Note that to use :py:class:`Cariolle` and :py:class:`Simotrostra`, the
+simotrostra package needs to be installed. It can be found here:
+https://gitlab.com/simple-stratospheric-chemistry/simotrostra
+
```

Comment:
Good idea to state this more clearly in the documentation!

One could even include this in an error message in the specific classes:
```py
try:
    import simotrostra
except ModuleNotFoundError:
    raise ModuleNotFoundError(
        "No module named 'simotrostra'. You need to install it in order to use this class.\n"
        "It can be found here:\n"
        "    https://gitlab.com/simple-stratospheric-chemistry/simotrostra"
```

Reply:
Good idea :)